app/javascript/packs/containers/DelegationContainer.jsx

```diff
@@ -39,7 +48,16 @@ class DelegationContainer extends React.Component {
   }
 
   handleSelect(value) {
```

Comment:
Maybe be explicit about _what_ user flow this belongs to, i.e. `handleExistingContactSelect` or `handleContactSelect` ? Makes it a little more clear as to not only what event this handles but why it handles it. Inline documentation also appreciated ☺️